docs/admin_guide.md

```diff
@@ -280,31 +307,64 @@ Description:               Run Compaction for given instant time
 * compaction run - Run Compaction for given instant time
 ```
 
-##### Up-Coming CLI for Compaction
-
-In the next release, more useful CLI to revert/repair compaction schedules will be added. Here is a preview of them:
+##### Validate Compaction
 
 Validating a compaction plan : Check if all the files necessary for compactions are present and are valid
 
 ```
-hoodie:trips->compaction validate --compactionInstant <instantId>
+hoodie:stock_ticks_mor->compaction validate --instant 20181005222611
```

Comment:
Should we add an example of what an invalid plan looks like and what error the user can expect ?

Reply:
Added